Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -0,0 +1,4381 @@
+
+    Args:
+        issues (List[Dict[str, Any]]): The issues that belong to the epic (not empty!).
+        epic_id_or_key (str): The epic id or key.
+        res (Dict[str, Any]): The raw response when calling the API to retrieve the epic's issues.
+
+    Returns:
+       CommandResults: CommandResults to return to XSOAR.
+    """
+    markdown_list = []
+    issues_list = []
+    if not issues:
+        return CommandResults(readable_output='No issues found to retrieve the epic, or board ID', raw_response=res)
+    for issue in issues:
+        markdown_dict, outputs_context_data = create_issue_md_and_outputs_dict(issue_data=issue)
+        markdown_list.append(markdown_dict)
+        issues_list.append(outputs_context_data)
+    outputs: Dict[str, Any] = {'Ticket': issues_list}
+    sprint = issues[0].get('fields', {}).get('sprint', {}) or {}
+    board_id = str(sprint.get(
+        'originBoardId', '')) or ''
+    outputs |= {'boardId': board_id} if board_id else {}
+    epic = issues[0].get('fields', {}).get('epic') or {}
+    epic_id = str(epic.get('id', '')) or ''
+    outputs |= {'epicId': epic_id} if epic_id else {}
+    return CommandResults(
+        outputs_prefix='Jira.EpicIssues',
+        outputs_key_field=['epicId', 'boardId'] if board_id else ['epicId'],
+        outputs=outputs,
+        readable_output=tableToMarkdown(name=f'Child Issues in epic {epic_id_or_key}', t=markdown_list),
+        raw_response=res
+    )
+
+
+def get_issue_link_types_command(client: JiraBaseClient, args: Dict[str, Any]) -> CommandResults:
+    """This command is in charge of returning the issue links between issues (Blocked by, Duplicates,...)
+
+    Args:
+        client (JiraBaseClient): The Jira client.
+        args (Dict[str, Any]):  The arguments supplied by the user.
+
+    Returns:
+        CommandResults: CommandResults to return to XSOAR.
+    """
+    res = client.get_issue_link_types()
+    issue_link_types = res.get('issueLinkTypes', [])
+    md_dict = [
+        {
+            'ID': issue_link_type.get('id', ''),
+            'Name': issue_link_type.get('name', ''),
+            'Inward': issue_link_type.get('inward', ''),
+            'Outward': issue_link_type.get('outward', ''),
+        }
+        for issue_link_type in issue_link_types
+    ]
+    return CommandResults(
+        outputs_prefix='Jira.IssueLinkType',
+        outputs=issue_link_types,
+        readable_output=tableToMarkdown(name='Issue Link Types', t=md_dict),
+        raw_response=res
+    )
+
+
+def link_issue_to_issue_command(client: JiraBaseClient, args: Dict[str, Any]) -> CommandResults:
+    """This command is in charge of adding an issue link between two issues.
+
+    Args:
+        client (JiraBaseClient): The Jira client.
+        args (Dict[str, Any]): The arguments supplied by the user.
+
+    Returns:
+        CommandResults: CommandResults to return to XSOAR.
+    """
+    outward_issue = args.get('outward_issue', '')
```

Comment:
maybe we should verify that  outward_issue and inward_issue are keys (not IDs) and raise exception otherwise.

Reply:
I added the support for both keys and IDs